CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-## [0.1.0] - 2021-09-29
+## [0.1.2] - 2021-11-16
```

Comment:
Based on the [Semver guide](https://semver.org/), since we are changing the behavior of the `set` method (which is potentially a breaking change), should this be a `major` bump in version instead of a `patch`?

Reply:
Oh yeah indeed, very good point!